src/overlays/actors/ovl_En_Ending_Hero6/z_en_ending_hero6.h

```diff
@@ -9,9 +9,18 @@ typedef void (*EnEndingHero6ActionFunc)(struct EnEndingHero6*, GlobalContext*);
 
 typedef struct EnEndingHero6 {
     /* 0x0000 */ Actor actor;
-    /* 0x0144 */ char unk_144[0x134];
+    /* 0x0144 */ SkelAnime skelAnime;
+    /* 0x0188 */ Vec3s limbDrawTable[20];
+    /* 0x0200 */ Vec3s transitionDrawTable[20];
```

Comment:
```suggestion
    /* 0x0188 */ Vec3s jointTable[20];
    /* 0x0200 */ Vec3s morphTable[20];
```
`limbDrawTable` and `transitionDrawTable` are legacy names that are currently being dropped for the more descriptive names `jointTable` and `morphTable` respectively. There's an open PR fixing all these names in the repo, so you can go ahead and use these updated names